students/ShinTran/lesson07/html_render.py

```diff
@@ -0,0 +1,134 @@
+'''
+Shin Tran
+Python 210
+Assignment 7
+'''
+
+#!/usr/bin/env python3
+
+# A class-based system for rendering html.
+
+
+# This is the framework for the base class
+class Element(object):
+
+    _Indent = '    '    # Default indent of 4 spaces
+    tag = 'html'        # Default tag name set to 'html'
+
+    # Initializes the program, if there's no content, returns an empty list
+    # the style parameter is for formatting a specific tag 
+    def __init__(self, content=None, **style):
```

Comment:
General convention is **kwargs for keyword arguments; `def __init__(self, content=None, **kwargs):`